plugins/source/pagerduty/codegen/recipes/templates/basic/resource.go.tpl

```diff
@@ -0,0 +1,11 @@
+// Code generated by codegen; DO NOT EDIT.
+
+package {{.SubService}}
```

Comment:
```suggestion
package {{.Service}}
```

Reply:
The `pagerduty` plugin doesn't really have a hierearchy of `service-subservice`, so I just used one of them randomly.

The difference seems small/interchangeble to me.